src/dev/impl/DevToys/Assets/Icons/XmlValidator.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Where is this SVG coming from? Is there a license we should pay attention?

Reply:
Uh, must admit that I did not really payed attention to the license of the icon - aaaand was unable to find the original source page again 😅 
That's why I just picked a new one with an appropriate license (no copyright restrictions at all 😁). 
Hope you find it to be as pretty as the original one!

Reply:
That's awesome ! :) Thank you very much!